server/app/views/applicant/EditTiClientView.java

```diff
@@ -214,16 +237,58 @@ private DivTag renderEditClientForm(
                 .withClasses("border", "border-gray-300", "shadow-md", "w-1/2", "mt-6"));
   }
 
+  private String getDefaultDob(Optional<ApplicantData> mayBeApplicantData) {
```

Comment:
praise: I think these helper methods are really useful to keep the form creation easy to read!